packages/codemirror-graphql/src/hint.js

```diff
@@ -80,3 +81,17 @@ CodeMirror.registerHelper('hint', 'graphql', (editor, options) => {
 
   return results;
 });
+
+function getType(schema, name) {
```

Comment:
Could the string `name` ever contain spaces? If so, it doesn't look like the code can handle this.

Reply:
`name` can contain spaces, but in that case, `undefined` is returned.

Reply:
`name` can contains spaces. If `name` contains spaces and `schema` contains the type of the `name`, this function returns that type. If `name` contains spaces and `schema` doesn't contain the type of the `name`, this function returns `undefined`.